js/index.js

```diff
@@ -1 +1,74 @@
 // Your code goes here
+
+const logoHeading = document.querySelector('h1');
+console.log(logoHeading);
+logoHeading.addEventListener('mouseover', event => {   
+event.target.style.color = 'yellow';
+setTimeout(function() {
+    event.target.style.color = "";
+  }, 1500);
+}, false);
+
+
+const buttons = document.querySelectorAll('.btn');
+buttons.forEach(button =>{
+button.addEventListener('click',event =>{
+event.target.style.backgroundColor = 'yellow';
+});
+});
+
+
+
+const dottedLines = document.getElementsByClassName('intro');
+// let dottedLines = document.getElementsByClassName('content-section inverse-content');
+// let dottedLines = document.getElementsByClassName('content-pickgit');
+dottedlines.addEventListener('scroll', event => {
```

Comment:
This doesn't work because of syntax error
```suggestion
dottedLines.addEventListener('scroll', event => {
```